File: commonLib/src/main/java/com/travel/commonlib/activity/city/Shards.java

package com.travel.commonlib.activity.city;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Shards {

    @SerializedName("total")
    @Expose
    private long total;
    @SerializedName("successful")
    @Expose
    private long successful;
    @SerializedName("skipped")
    @Expose
    private long skipped;
    @SerializedName("failed")
    @Expose
    private long failed;

    public long getTotal() {
        return total;
    }

    public void setTotal(long total) {
        this.total = total;
    }

    public long getSuccessful() {
        return successful;
    }

    public void setSuccessful(long successful) {
        this.successful = successful;
    }

    public long getSkipped() {
        return skipped;
    }

    public void setSkipped(long skipped) {
        this.skipped = skipped;
    }

    public long getFailed() {
        return failed;
    }

    public void setFailed(long failed) {
        this.failed = failed;
    }

}

File: commonLib/src/main/java/com/travel/commonlib/hotel/elastic/Shards.kt
package com.travel.commonlib.hotel.elastic

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Shards : Serializable {
    @SerializedName("total")
    @Expose
    var total = 0

    @SerializedName("successful")
    @Expose
    var successful = 0

    @SerializedName("skipped")
    @Expose
    var skipped = 0

    @SerializedName("failed")
    @Expose
    var failed = 0
}